command.py

```diff
@@ -0,0 +1,244 @@
+        embed.add_field(name=f"チャンネル数({len(guild.channels)})",value=f"**{len(guild.text_channels)}** テキストチャンネル\n**{len(guild.voice_channels)}** ボイスチャンネル")
+        embed.add_field(name=f"ロール({len(guild.roles)})",value="".join([rlb]),inline=False)
+        embed.set_author(icon_url=interaction.user.avatar.url,name=interaction.user.global_name)
+        embed.set_footer(text=guild.name)
+
+        await interaction.response.send_message(embed=embed)
+
+    @app_commands.command(description="指定したユーザーの情報を表示します")
+    @app_commands.describe(user="情報を表示したいユーザーを指定します")
+    async def info(self, interaction: discord.Interaction, user: discord.Member = None):
+        """
+        A command to show information of a specified user
+        """
+        guild=interaction.guild
+
+        if user is None:
+            role_list=[]
+            for role in interaction.user.roles:
+                if role.name != "@everyone":
+                    role_list.append(role.mention)
+            rlb=" | ".join(role_list)
+            embed=discord.Embed(title=f"{interaction.user.name} - Info",color=discord.Color.blue(),
+                                timestamp=datetime.datetime.now())
+            embed.add_field(name="アカウント作成日",value=f"<t:{int(time.mktime(interaction.user.created_at.timetuple()))}:R> | <t:{int(time.mktime(interaction.user.created_at.timetuple()))}:D>")
+            embed.add_field(name="アカウント参加日",value=f"<t:{int(time.mktime(interaction.user.joined_at.timetuple()))}:R> | <t:{int(time.mktime(interaction.user.joined_at.timetuple()))}:D>")
+            embed.add_field(name="ユーザーID",value=interaction.user.id,inline=False)
+            embed.add_field(name="アイコンURL",value=f"[ユーザーアバター URL]({interaction.user.avatar.url})")
+            embed.add_field(name=f"ロール({len(interaction.user.roles)})",
+                            value="".join([rlb]),inline=False)
+            embed.set_author(icon_url=interaction.user.avatar.url,name=interaction.user.global_name)
+            embed.set_footer(text=guild.name)
+            embed.set_thumbnail(url=interaction.user.avatar.url)
+
+            await interaction.response.send_message(embed=embed)
+
+        else:
+            role_list=[]
+            for role in user.roles:
+                if role.name != "@everyone":
+                    role_list.append(role.mention)
+            rlb=" | ".join(role_list)
+            embed=discord.Embed(title=f"{user.name} - Info",color=discord.Color.blue(),
+                                timestamp=datetime.datetime.now())
+            embed.add_field(name="アカウント作成日",
+                            value=f"<t:{int(time.mktime(user.created_at.timetuple()))}:R> | <t:{int(time.mktime(user.created_at.timetuple()))}:D>")
+            embed.add_field(name="アカウント参加日",value=f"<t:{int(time.mktime(user.joined_at.timetuple()))}:R> | <t:{int(time.mktime(user.joined_at.timetuple()))}:D>")
```

Comment:
This is also the same.
```suggestion
            created_timetuple = interaction.user.created_at.timetuple()
            embed.add_field(name="アカウント作成日",
                            value=" | ".join([f"<t:{int(time.mktime(created_timetuple))}:R>",
                                              f"<t:{int(time.mktime(created_timetuple))}:D>"])
                            )
            joined_timetuple = interaction.user.joined_at.timetuple()
            embed.add_field(name="アカウント参加日",
                            value=" | ".join([f"<t:{int(time.mktime(joined_timetuple))}:R>",
                                              f"<t:{int(time.mktime(joined_timetuple))}:D>"])
                            )
```